### ISSUE TITLE
GraphQL Page API - Implement `ViewAs` field 

### ISSUE THREAD

**Issue by dsilvam** [dotCMS/core#19108]
Given query that includes the `viewAs` field with all its available sub-fields: 

```
{
  page(url:"/destinations/costa-rica", pageMode: "live", languageId: "1") {
     viewAs {
      persona {
        name
        description
        photo {
          idPath
        }
        keyTag
        tags
      }
      visitor {
        persona {
          name
          description
          photo {
            idPath
          }
          keyTag
          tags
        }
        device
        isNew
        tags {
          tag
          count
        }
        userAgent {
          operatingSystem
          browser
          id
          browserVersion {
            version
            majorVersion
            minorVersion
          }
        }
        personas {
          persona 
          count
        }
      }
      mode
      language {
        id
        country
        countryCode
        language
        languageCode
      }
    }
  }
}
```

The response needs to look like this: 
```
{
    "data": {
        "page": {
            "viewAs": {
                "persona": {
                    "name": "Winter Enthusiast",
                    "description": "People who are passionate about winter sports.",
                    "photo": {
                        "idPath": "/dA/792c7c9f6b/photo/mountain-persona.jpg?language_id=1"
                    },
                    "keyTag": "WinterEnthusiast",
                    "tags": [
                        "snowmobile",
                        "snowboarding",
                        "skiing"
                    ]
                },
                "visitor": {
                    "persona": {
                        "name": "Winter Enthusiast",
                        "description": "People who are passionate about winter sports.",
                        "photo": {
                            "idPath": "/dA/792c7c9f6b/photo/mountain-persona.jpg?language_id=1"
                        },
                        "keyTag": "WinterEnthusiast",
                        "tags": [
                            "snowmobile",
                            "snowboarding",
                            "skiing"
                        ]
                    },
                    "device": "COMPUTER",
                    "isNew": true,
                    "tags": [
                        {
                            "tag": "snowmobile",
                            "count": 25
                        },
                        {
                            "tag": "skiing",
                            "count": 25
                        },
                        {
                            "tag": "snowboarding",
                            "count": 25
                        },
                        {
                            "tag": "diving",
                            "count": 1
                        }
                    ],
                    "userAgent": {
                        "operatingSystem": "MAC_OS_X",
                        "browser": "CHROME8",
                        "id": 50990853,
                        "browserVersion": {
                            "version": "84.0.4147.125",
                            "majorVersion": "84",
                            "minorVersion": "0"
                        }
                    },
                    "personas": [
                        {
                            "persona": "WinterEnthusiast",
                            "count": 1
                        }
                    ]
                },
                "mode": "LIVE",
                "language": {
                    "id": 1,
                    "country": "United States",
                    "countryCode": "US",
                    "language": "English",
                    "languageCode": "en"
                }
            }
        }
    }
}
```

**Comment by dsilvam** [comment]
PR: https://github.com/dotCMS/core/pull/19119

**Comment by dsilvam** [comment]
Note to QA: Postman test is `GraphQL.PageAPI.GivenRequestByURI_ViewAsFieldContainsAllFields`

**Comment by freddyucv** [comment]
**Passed Internal QA**: Check postman test

**Comment by wezell** [comment]
Visitor geo is hardcoded to "ipAddress"

**Comment by wezell** [comment]
fixed in : #19159

**Comment by bryanboza** [comment]
fixed, tested on release-5.3.8 // Postgres // FF
